packages/block-editor/README.md

```diff
@@ -558,6 +558,26 @@ _Related_
 
 Undocumented declaration.
 
+<a name="useBlockProps" href="#useBlockProps">#</a> **useBlockProps**
```

Comment:
Noting that `useBlockProps.save` isn't documented. I guess it's too late to expose it under its own name.

Reply:
We discussed this and we preferred to use that notation because we'll have the same thing for innerblocks, richtext, and each one of this have a version for edit and for save.

And we plan to document this properly see the PR description here https://github.com/WordPress/gutenberg/pull/26100

Reply:
Cool, makes sense to follow the same pattern, although it isn't ideal moving forward :)

Reply:
What's not ideal moving forward?

Reply:
The dot notation that takes advantage of the fact that a function is also an object in JS.